src/vorta/views/diff_result.py

```diff
@@ -97,7 +98,9 @@ def __init__(self, archive_newer, archive_older, model: 'DiffTree'):
         self.archiveNameLabel_1.setText(f'{archive_newer.name}')
         self.archiveNameLabel_2.setText(f'{archive_older.name}')
 
+        diff_result_display_mode = SettingsModel.get(key='diff_files_display_mode').str_value
         self.comboBoxDisplayMode.currentIndexChanged.connect(self.change_display_mode)
+        self.comboBoxDisplayMode.setCurrentIndex(int(diff_result_display_mode))
```

Comment:
```suggestion
        self.comboBoxDisplayMode.currentIndexChanged.connect(self.change_display_mode)
        self.comboBoxDisplayMode.setCurrentIndex(int(SettingsModel.get(key='diff_files_display_mode').str_value))
```

This or move line 101 down one line.